Source/WebCore/platform/graphics/cocoa/FontFamilySpecificationCoreText.cpp

```diff
@@ -57,7 +58,8 @@ FontRanges FontFamilySpecificationCoreText::fontRanges(const FontDescription& fo
 
         return makeUnique<FontPlatformData>(font.get(), size, false, syntheticOblique, fontDescription.orientation(), fontDescription.widthVariant(), fontDescription.textRenderingMode());
     });
-    return FontRanges(FontCache::forCurrentThread().fontForPlatformData(fontPlatformData));
+
+    return FontRanges(FontCache::forCurrentThread().fontForPlatformData(FontPlatformData::fontPlatformDataWithAdjustedSize(WTFMove(platformData), fontDescription)));
```

Comment:
The explicit conversion to `FontRanges` was not needed before and is still not needed after.

Reply:
Same as above, I think we shouldn't make 2 FontPlatformDatas here.

Reply:
FontRanges has a ctor that takes RefPtr<Font> (instead of Ref<Font>) but it is explicit.